themes/openy_themes/openy_rose/openy_rose.theme

```diff
@@ -494,5 +494,15 @@ function openy_rose_preprocess_paragraph_social_list(&$variables) {
  */
 function openy_rose_preprocess(array &$variables) {
   $variables['base_path'] = base_path();
-  $variables['directory'] =  drupal_get_path('theme', 'openy_rose');
+  $variables['directory'] = drupal_get_path('theme', 'openy_rose');
+}
+
+/**
+ * Implements hook_preprocess_video_embed_iframe().
+ */
+function openy_rose_preprocess_video_embed_iframe(array &$variables) {
```

Comment:
I think maybe it will be better to move this function to https://github.com/open-y-subprojects/openy_features/tree/main/openy_media/modules/openy_media_video instead of repeating it 3 times?

cc @podarok @aleevas

Reply:
We are going to decouple themes in Q4.
If we think all themes should have a dependency from openy_media_video - I'm fine with moving code there @hamrant